homeassistant/components/fritz/__init__.py

```diff
@@ -79,3 +81,11 @@ async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     await async_unload_services(hass)
 
     return unload_ok
+
+
+async def update_listener(hass: HomeAssistant, entry: ConfigEntry):
+    """Update when config_entry options update."""
+    fritzbox_tools: FritzBoxTools = hass.data[DOMAIN][entry.entry_id]
+
+    if fritzbox_tools._options:
```

Comment:
```suggestion
    if entry.options:
```

Reply:
Not sure if its worth checking though

Reply:
So you suggest just a bare update ?

Reply:
Is there any other time we update the config entry that we would not want a reload?

Reply:
Not now that we have only 1 config option